docs/integration.md

```diff
@@ -42,24 +47,67 @@ Note: Need to use **@import CleverTapSDK;** instead of **#import <CleverTap-iOS-
         // example:
         packages.add(new CleverTapPackage());// only needed when not auto-linking
         return packages;
+        
+    }
+    ```
 
+3. Initialise Clevertap ReactNative Integration - This adds support for `ClevertapPushNotiificationClicked` from killed state and registers the `ActivityLifecycleCallback`
+- <a name="step3a"></a>From clevertap-react-native **v3.0.0** onwards, if developers **don't** want their `Application` class to extend `CleverTapApplication`, they should call `CleverTapRnAPI.initReactNativeIntegration(this)` and `ActivityLifecycleCallback.register(this)` from the `onCreate()` to support Push Notification click callback in killed state.
+
+    ```java
+    import com.clevertap.react.CleverTapRnAPI;
+    import com.clevertap.android.sdk.ActivityLifecycleCallback;
+    import com.clevertap.android.sdk.CleverTapAPI;
+    import com.clevertap.android.sdk.CleverTapAPI.LogLevel;
     // ...
 
-    // add onCreate() override
-    @Override
-    public void onCreate() {
-	    // Register the CleverTap ActivityLifecycleCallback; before calling super
-        ActivityLifecycleCallback.register(this);	
-        super.onCreate();
-        // Init CleverTapSDK React Native integration
-        CleverTapRnAPI.initReactNativeIntegration(this);
-    }
+        public class MainApplication implements ActivityLifecycleCallbacks, ReactApplication {
```

Comment:
Is it required for this class to implement ActivityLifecycleCallbacks ?

Reply:
Updated thanks